geoh5py/objects/grid_object.py

```diff
@@ -55,6 +56,7 @@ def copy(
         copy_children: bool = True,
         clear_cache: bool = False,
         mask: np.ndarray | None = None,
+        copy_property_groups: bool = True,
```

Comment:
Let's not just modify this class. This would technically have to be pushed to the base class.